apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -482,15 +540,32 @@ func (s *modelManagerSuite) testCreateModelDefaultCredentialAdmin(c *gc.C, owner
 }
 
 func (s *modelManagerSuite) TestCreateModelEmptyCredentialNonAdmin(c *gc.C) {
-	defer s.setUpMocks(c).Finish()
+	ctrl := s.setUpMocks(c)
+	defer ctrl.Finish()
 	s.setUpAPI(c)
-	s.accessService.EXPECT().GetModelUsers(gomock.Any(), user.AdminUserName, gomock.Any()).Return(
-		[]access.ModelUserInfo{{
-			Name:        usertesting.GenNewName(c, "user-bob"),
-			DisplayName: "Bob",
-			Access:      permission.ReadAccess,
-		}}, nil,
+	bobName := usertesting.GenNewName(c, "bob")
+	bobUUID := usertesting.GenUserUUID(c)
+	modelUUID := modeltesting.GenModelUUID(c)
+
+	s.modelService.EXPECT().DefaultModelCloudNameAndCredential(
+		gomock.Any()).Return("dummy", credential.Key{}, nil)
+	s.accessService.EXPECT().GetUserByName(
+		gomock.Any(), bobName,
+	).Return(user.User{UUID: bobUUID}, nil)
+
+	s.modelService.EXPECT().CreateModel(gomock.Any(), model.ModelCreationArgs{
```

Comment:
So why can't this one be rolled into the same `expectCreateModel` function?

Reply:
(I'm going to make these changes in the other PR and close this one.)
One calls the last method slightly differently because its not an admin user. I would have had to add more parameters to the expect function and didn't really want to here.